schemas/FeatureManagement.Dotnet.v1.0.0.schema.json

```
@@ -0,0 +1,102 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "A.NET Feature Management Configuration",
+  "required": [],
+  "patternProperties": {
+    "^[^:]*$": {
+      "description": "Declares a feature flag.",
+      "anyOf": [
+        {
+          "type": "boolean",
+          "title": "On/Off Feature Flag",
+          "description": "A feature flag that always returns the same value."
+        },
+        {
+          "type": "object",
+          "title": "Conditional Feature Flag",
+          "description": "A feature flag which value is dynamic based on a set of feature filters",
+          "required": [
+            "EnabledFor"
+          ],
+          "properties": {
+            "RequirementType": {
+              "type": "string",
+              "title": "Requirement Type",
+              "description": "Determines whether any or all registered feature filters must be enabled for the feature to be considered enabled.",
+              "enum": [
+                "Any",
+                "All"
+              ],
+              "default": "Any"
+            },
+            "EnabledFor": {
+              "oneOf": [
+                {
+                  "type": "array",
+                  "title": "Feature Filter Collection",
+                  "description": "Feature Filters that must be evaluated as true for the feature to be considered enabled.",
```

This wording is a listtle sensitive, sense for default requirement type only a single filter needs to be true. I would say instead

```suggestion
                  "description": "Feature filters that are evaluated to conditionally enable the flag.",
```